src/test/spec/unified_runner/entity.rs

```diff
@@ -25,6 +31,12 @@ pub struct ClientEntity {
     pub ignore_command_names: Option<Vec<String>>,
 }
 
+#[derive(Clone, Debug)]
+pub struct SessionEntity {
+    pub lsid: Document,
+    pub client_session: Option<Box<ClientSession>>,
```

Comment:
I'm not sure why the ClientSession is boxed here?

Reply:
Clippy errors without the box, since the struct is too large otherwise.
Should I suppress the linter error instead?

Reply:
Oh in that case its probably just fine to keep the `Box` then imo.